src/main/java/tutorlink/commons/Commons.java

```diff
@@ -49,4 +49,47 @@ public class Commons {
 
     //Exit
     public static final String EXIT = "Goodbye! See you soon!";
+
+    //Help
+    public static final String HELP_MESSAGE = """
+            -------------------   List of Commands   --------------------
```

Comment:
Perhaps the break line can be removed from the HELP_MESSAGE string?
Since it will be passed into ui.displayResult(), which will print break lines at the start and end again.